src/literature/components/PublicationInfo.jsx

```diff
@@ -14,15 +28,9 @@ class PublicationInfo extends Component {
             <span> {info.get('journal_volume')}</span>
           )}
           {info.has('year') && <span> ({info.get('year')})</span>}
-          {info.has('page_start') &&
-            info.get('page_end') && (
-              <span>
-                {' '}
-                {info.get('page_start')}-{info.get('page_end')}
-              </span>
-            )}
-          {info.has('page_start') && <span> {info.get('page_start')}</span>}
-          {info.has('artid') && <span> {info.get('artid')}</span>}
+          {journalIssue && <span> {journalIssue}</span>}
+          {this.getPageOrArtidDisplay() && journalIssue && <span>,</span>}
```

Comment:
`const pageOrArtidDisplay = this.getPageOrArtidDisplay()` above, so that method is called just once.